.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       REACT_APP_STRIPE_PUBLISHABLE_KEY: '${{ secrets.STRIPE_PUBLISHABLE_KEY }}'
       REACT_APP_BUGSNAG_API_KEY: '${{ secrets.BUGSNAG_API_KEY }}'
       REACT_APP_LOGROCKET_APP_ID: '${{ secrets.LOGROCKET_APP_ID }}'
+      REACT_APP_AGORA_APP_ID: '${{ secrets.AGORA_APP_ID }}'
+      REACT_APP_AGORA_CHANNEL: '${{ secrets.AGORA_CHANNEL }}'
+      REACT_APP_AGORA_TOKEN: '${{ secrets.AGORA_TOKEN }}'
```

Comment:
We won't need any of these here when using the real token fetching from backend method I wrote.

```suggestion
```

Reply:
@0xdevalias for hooks working (src/hooks/video/agora/index.ts) I need an App Id and Channel Name in addition to token. If I'll delete now these ENV from file, so agora won't work too.

Reply:
Look at the implementation on my draft PR where I wire everything up. It also contains some tweaks to the backend functionality to support getting the app id.